example/PaymentDao.java

```diff
@@ -0,0 +1,50 @@
+public class PaymentDao {
+
+    private static final Logger logger = LoggerFactory.getLogger(PaymentDao.class);
+
+    private final Session session;
+
+    public PaymentDao(Session session) {
+        this.session = session;
+    }
+
+    /**
+     * Saves payment in the database.
```

Comment:
Update documentation to include @throws IllegalArgumentException when payment is null.